modules/oe_translation_cdt/tests/src/Kernel/TranslationRequestUpdaterTest.php

```diff
@@ -139,9 +139,9 @@ public function testUpdateFromTranslationResponse(): void {
       'Manually updated the status.' .
       'Updated request_status field from Requested to Translated.' .
       'The following languages are updated: fr (Requested =&gt; Review).' .
-      'Updated priority field from PRIO1 to PRIO2.' .
       'Updated comments field to TEST.' .
       'Updated phone_number field from 999999999 to 123456789.' .
+      'Updated priority field from PRIO1 to PRIO2.' .
```

Comment:
we can cover this a bit better with testing by not setting the job summary. Perhaps a translation response with only minimal or required fields, assert, new response with all fields, assert